packages/sit-onyx/src/components/OnyxMenuItem/OnyxMenuItem.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
```

Comment:
Per my understanding, the `OnyxMenuItem` can do everything that the `OnyxNavItem` can, do, correct?

Can we use the `OnyxNavItem` as a thin wrapper around `OnyxNavItem`, that just restricts the props a bit?
As far as I remember, the only difference was that the `danger` color should not be allowed in the OnyxNavItem?

Reply:
Yes you are right. The only difference is the color. Did you mean to use the OnyxMenuItem as a thin wrapper around OnuxNavItem?

Reply:
The `OnyxMenuItem` is more powerful. When we wrap it inside the `OnyxNavItem`, we can limit the props to what we want :)